File: examples/functions/python/nombre/handler.py
import json


def handler(event):
    query = event.get("query") or {}
    name = query.get("name", "world")

    return {
        "status": 200,
        "headers": {"Content-Type": "application/json"},
        "body": json.dumps(
            {
                "runtime": "python",
                "function": "hello",
                "hello": name,
                "request_id": event.get("id"),
            },
            separators=(",", ":"),
        ),
    }

File: examples/functions/assets-worker-first/rust/hello/handler.rs
use serde_json::{json, Value};

pub fn handler(_event: Value) -> Value {
    json!({
        "status": 200,
        "headers": {
            "Content-Type": "text/plain; charset=utf-8"
        },
        "body": "hello from rust runtime"
    })
}
